security/x509.go

```diff
@@ -91,7 +93,7 @@ func newTemplate(commonName string) (*x509.Certificate, error) {
 			Organization: []string{"Cockroach"},
 			CommonName:   commonName,
 		},
-		NotBefore: notBefore,
+		NotBefore: notBefore.Add(validFrom),
```

Comment:
I liked the visual of having both next to each other. But I can if you really insist.